packages/sdk-connector/package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "@towns-protocol/sdk-connector",
```

Comment:
Ill probably change the name to rpc-connector instead
`@towns-protocol/rpc-connector` reads way better